pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -773,6 +773,15 @@ def test_aggregate_mixed_types():
     tm.assert_frame_equal(result, expected)
 
 
+def test_aggregate_categorical_lost_index():
```

Comment:
can you parameterize this for both min and max (expected is the same)

Reply:
Oh thanks. Is done